src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,103 @@
 import React from 'react';
 
-const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+
+const LandingPage = (
+  props = {
```

Comment:
This is a _slightly_ funky way to do props defaulting that works here, but might not do what you intend in many situations.  This code says "If absolutely no props object is supplied, use this one".  It won't give you a `tabs` value if a `props` parameter is supplied that doesn't contain one.